To: vim_dev@googlegroups.com
Subject: Patch 7.3.149
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.149
Problem:    The cursor disappears after the processing of the 'setDot'
	    netbeans command when vim runs in a terminal.
Solution:   Show the cursor after a screen update. (Xavier de Gaye, 2011
Files:	    src/netbeans.c


*** ../vim-7.3.148/src/netbeans.c	2011-01-04 18:11:39.000000000 +0100
--- src/netbeans.c	2011-04-01 15:33:49.000000000 +0200
***************
*** 191,196 ****
--- 191,197 ----
      changed_window_setting();
      update_screen(CLEAR);
      setcursor();
+     cursor_on();
      out_flush();
  #ifdef FEAT_GUI
      if (gui.in_use)
***************
*** 2248,2253 ****
--- 2249,2255 ----
  	    update_topline();		/* scroll to show the line */
  	    update_screen(VALID);
  	    setcursor();
+ 	    cursor_on();
  	    out_flush();
  #ifdef FEAT_GUI
  	    if (gui.in_use)
***************
*** 2642,2647 ****
--- 2644,2650 ----
      {
  	update_screen(NOT_VALID);
  	setcursor();
+ 	cursor_on();
  	out_flush();
  #ifdef FEAT_GUI
  	if (gui.in_use)
***************
*** 3008,3013 ****
--- 3011,3017 ----
      changed_window_setting();
      update_screen(CLEAR);
      setcursor();
+     cursor_on();
      out_flush();
  #ifdef FEAT_GUI
      if (gui.in_use)
*** ../vim-7.3.148/src/version.c	2011-04-01 14:44:54.000000000 +0200
--- src/version.c	2011-04-01 15:33:21.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     149,
  /**/

-- 
ARTHUR:       You are indeed brave Sir knight, but the fight is mine.
BLACK KNIGHT: Had enough?
ARTHUR:       You stupid bastard.  You havn't got any arms left.
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
